.github/workflows/smoke-test.yaml

```diff
@@ -0,0 +1,17 @@
+name: Action Smoke Tests
+on: workflow_dispatch
+jobs:
+  run-action:
+    name: Run GH Action
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout Repository
+      - uses: ./
+        name: Run GH Action on ${{ github.ref }}
+        env:
+          CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
+          GITHUB_EVENT_PATH: .github/.vib/github-event.json
```

Comment:
This will make every pull request to trigger a build from the same git ref. I don't think this is what you want here.

Reply:
Actually that was my intention. Since we want to validate the action and not the pipeline itself, I thought it would be useful exercising the same pipeline every time. This way, if the CI step fails at some point, we cannot blame the application under validation but either the GH action or VIB

Reply:
I see what you mean. Right, that should work.